### ISSUE TITLE
Error when using NPM module with Rollup worker bundled

### ISSUE THREAD

**Issue by a-type** [GoogleChromeLabs/worker-plugin#93]
I'm using this plugin with a Create-React-App based project (utilizing react-app-rewired to inject it into the Webpack plugin). So far that was working great.

I went to add [use-cannon](https://github.com/pmndrs/use-cannon), however, and encountered a strange build error:

```
Failed to compile.

./node_modules/use-cannon/dist/Provider-f05cb643.js
Module parse failed: Cannot read property 'length' of undefined
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
TypeError: Cannot read property 'length' of undefined
```

Ultimately I've narrowed it down to worker-plugin. If I run my app without `react-app-rewired` (i.e. not adding the plugin), use-cannon works fine.

A minimal reproduction repo lives [here](https://github.com/a-type/worker-cannon-repro). The error shows up when starting the project. You can switch the `npm start` script back to `react-scripts start` to see that it runs fine without worker-plugin.

There's probably something specifically off about the way worker-plugin tries to operate on the worker that's already taken care of within the built bundle of use-cannon.

Possible solutions:

1. Add a way to omit node_modules dependencies from worker-plugin's processing
2. Exclude specific NPM dependencies by name
3. Make whatever processing is happening more fault-tolerant and just have it ignore the file if this error comes up

**Comment by lucas-janon** [comment]
For people searching for a temporary solution: downgrading to 4.0.3 worked (no pun intended) for me

**Comment by ezforw** [comment]
Facing the same problem as stated in https://github.com/auth0/auth0-angular/issues/54

The downgrade didn't work for me unfortunately

**Comment by ezforw** [comment]
BTW, it seems it's also affecting Angular as well, the issue can be reproduced here https://github.com/angular/angular-cli/issues/19884

**Comment by developit** [comment]
This should be fixed by #94.
